DB/GADPerson.m

```diff
@@ -95,6 +101,50 @@ -(GADPerson *)initWithDictionary: (NSDictionary *) dict {
     return people;
 }
 
++ (void) fetchPersonInfoWithCriteria:(NSDictionary*)crit andUsername:(NSString*)usrname andPassword:(NSString*) pw completionHandler:(void(^_Nonnull)(NSArray<GADPerson *> *))completion {
+
+    NSMutableArray * queryItems = [NSMutableArray array];
+    for (NSString * key in crit){
+        [queryItems addObject:[NSURLQueryItem queryItemWithName:key value:crit[key]]];
+    }
+    NSURLComponents *components = [NSURLComponents componentsWithString:@"https://itwebappstest.grinnell.edu/DotNet/WebServices/api/db"];
+    components.queryItems=queryItems;
+    NSURL *url = components.URL;
+    
+    NSString *Post = [NSString stringWithFormat:@"{'un':'%@', 'pw':'%@'}",usrname,pw];
```

Comment:
We use camelcase with a leading lowercase for variable names. Could we call this the `authenticationString` instead?